packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -565,3 +563,42 @@ You can create Object Types in three ways:
     ```
     > This method provides easy use and is ideal for smaller projects.
 
+#### Query
+
+Every schema requires a top level Query type. By default, the schema will look
+for the `Object Type` named `Query`. To add fields for these queries, we can 
+simply run the `addQuery` function to add to the schema's `Query` type.
```

Comment:
nit: maybe include a 1-liner that describes the behaviour of having a query type rather than only describe what's possible.